src/api/ErrorHandler.ts

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import type { ApiError } from "@/api/types";
+
+export const handleApiError = (
+  error: unknown,
+  fallbackMessage = "알 수 없는 오류가 발생했습니다",
+): never => {
+  if (axios.isAxiosError(error)) {
```

Comment:
```suggestion
if (axios.isAxiosError<BaseResponse<ApiError>>(error)) {
```
isAxiosError에 타입을 주입할 수 있습니다! 이러면 밑에 error.response.data 에 타입을 단언하지 않으셔도 될거 같아요!

Reply:
항상 as로 타입 단언을 했었는데 이런 식으로 쓸 수 있었네요!